internal/agent-api/types.go

```diff
@@ -71,32 +75,39 @@ type DeployRequest struct {
 	Errors []error `json:"errors,omitempty"`
 }
 
-func (w *DeployRequest) Validate() bool {
-	w.Errors = make([]error, 0)
+// Returns true if the run request supports trigger subjects
+func (request *DeployRequest) SupportsTriggerSubjects() bool {
+	return (strings.EqualFold(*request.WorkloadType, "v8") ||
+		strings.EqualFold(*request.WorkloadType, "wasm")) &&
+		len(request.TriggerSubjects) > 0
+}
+
+func (r *DeployRequest) Validate() bool {
+	r.Errors = make([]error, 0)
 
-	if w.WorkloadName == nil {
-		w.Errors = append(w.Errors, errors.New("workload name is required"))
+	if r.WorkloadName == nil {
+		r.Errors = append(r.Errors, errors.New("workload name is required"))
 	}
 
 	// FIXME-- this should be provided in the request
```

Comment:
I added that hash and bytes in another PR.  This comment may not be neccessary anymore

Reply:
https://github.com/synadia-io/nex/blob/b24398f5c4ad8f7d755756382701c504c2592ff6/internal/node/payload_cache.go#L96